osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -241,10 +303,31 @@ public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
             switch (e.Action)
             {
                 case GlobalAction.Back:
+                    PreferKeyboardNavigation();
                     return goBack();
 
                 case GlobalAction.Select:
-                    logo?.TriggerClick();
+                case GlobalAction.SelectNext:
+                    PreferKeyboardNavigation();
+                    CurrentSelection?.TriggerClick();
+                    // keyboard navgiation into submenu
+                    if (State != ButtonSystemState.EnteringMode)
+                        SelectionIndex = 1; // 1 to skip the back/settings btn
+                    return true;
+
+                case GlobalAction.SelectPrevious:
+                    PreferKeyboardNavigation();
+                    SelectUp();
+                    return true;
+
+                case GlobalAction.SelectNextGroup:
```

Comment:
Really not sure about hooking this keybinding up to these actions. `Select{Previous,Next}` are okay, but there is no grouping to be had here in sight.

Reply:
you are right, this was actually more of a leftover from my previous implementation where (down and right) and (up and left) were pairs of the same action. I think for simple keyboard navigation the current approach is good, but actually broke my taiko use-case which I initially made this for without even noticing.

Having a new set of definable buttons (FocusNext, FocusPrevious, EnterFocus) for this navigation would make it much more understandable as well as opening up the opportunity to implement this in other views as well.

Reply:
ok so after removing SelectPrevious and SelectNext, suddenly `Select{Previous,Next}Group` made much more sense in the code again and I decided to not add any new bindings here, and instead promote reusing next/previous group for focus group change. Will push in a sec